server/routes/user.js

```diff
@@ -23,16 +23,21 @@ router.get("/skills", getUsersBySkill);
 router.put("/acc", updateUserAcc);
 
 //Update user info/data
-router.put("/info", updateUserInfo);
+router.put("/info", ensureLoggedIn, updateUserInfo);
 
 //Update user photo
-router.put("/photo", upload.single("profilePic"), updateUserPhoto);
+router.put(
+  "/photo",
+  ensureLoggedIn,
+  upload.single("profilePic"),
+  updateUserPhoto
+);
 
 //Update user calendly link
 router.put("/calendar", setUserCalendlyLink);
```

Comment:
Hey @Dnhem  all looks good to me. Could you add the ensureLoggedIn to the Calendar route and modal too, please? We'd want to keep it protected so only a logged in verified user can update their Calendly link.

Reply:
No worries, I've done it 👍🏻! Fixed the BE Route protection on the photo modal too.